build.gradle

```diff
@@ -63,21 +65,22 @@ subprojects {
     // Required due to strictness of the Android developer tools.  See lint.xml for full details.
     lintOptions {
       lintConfig rootProject.file('gradle/lint.xml')
+      abortOnError false
```

Comment:
Do we really want to disable `abortOnError`?  I thought it would be a useful mechanism to keep us honest (and potentially catch some classes of bug).

If there's warnings/errors that need to be ignored, we can add specific exclusions (ideally each one documented) in https://github.com/cyclestreets/android/blob/master/gradle/lint.xml...

Reply:
In the long term no.  However, due to a change in the way permissions are now handled, the linter wants us to put security exception handlers around the places where we use the location manager.

We should do this, but it needs a little more thinking about that simply slapping in a try catch block.  For example, if we can't GPS then liveride should be completely disabled.  

Rather than do that in a hurry, I decided making the linter a bit less strict was the expedient thing to do.

Reply:
That rationale makes perfect sense.  However, rather than disabling `abortOnError` globally, we could instead choose to ignore the specific problems.  If you add the following snippet into `lint.xml`, it does the trick (I tested locally).

```xml
    <issue id="MissingPermission">
        <!-- TODO: https://github.com/cyclestreets/android/issues/204 -->
        <ignore path="**/RecordingService.java" />
        <ignore path="**/LiveRideService.java" />
    </issue>
```

Would you be happy to make this change instead?

Reply:
In fact; I'll merge your PR and raise this as a mini follow-up, to save time 😃